--- a/galaxy-central/lib/galaxy/datatypes/data.py	2021-01-08 17:02:57.747469477 +0000
+++ b/galaxy-central/lib/galaxy/datatypes/data.py	2021-01-08 17:03:23.987603597 +0000
@@ -435,7 +435,7 @@
         from galaxy import datatypes  # DBTODO REMOVE THIS AT REFACTOR
         if to_ext or isinstance(data.datatype, datatypes.binary.Binary):  # Saving the file, or binary file
             if data.extension in composite_extensions:
-                return self._archive_composite_dataset(trans, data, do_action=kwd.get('do_action'))
+                return self._archive_composite_dataset(trans, data, do_action=kwd.get('do_action','zip'))
             else:
                 trans.response.headers['Content-Length'] = int(os.stat(data.file_name).st_size)
                 filename = self._download_filename(data, to_ext, hdca=kwd.get("hdca", None), element_identifier=kwd.get("element_identifier", None))
